src/vuex/modules/pushNotifications.js

```diff
@@ -57,22 +67,27 @@ const askPermissionAndRegister = async (activeNetworks, apollo) => {
           const token = await messaging.getToken()
           await registerDevice(token, activeNetworks, apollo)
         })
-        .catch(error =>
-          console.log(
-            "bug FCM throws error while deleting token on first refresh",
-            error
-          )
-        )
+        .catch(() => {
+          localStorage.setItem("registration-push-notifications", "blocked")
```

Comment:
is the only error that can happen, that the user blocked the notification?

Reply:
I made the exact same comment on Slack! 🤣  amazing

Reply:
![neyo-1oC7B5v1VzaxJC9KeQ](https://media.giphy.com/media/4NgzOc3sZRHeIWkd3J/giphy.gif)

Reply:
hahahah we already think the same! dangerous :D

Reply:
xD

Reply:
Sorry to interrupt this ET party 😄 

After some deep deep googling I found that you can check for the code:
1. error.code = blocked => set localstorage
2. other errors? console.error

Good catch to improve error catching, thanks guys! 👍